content/academy/apify_platform/actors.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Actors
+description: What is an actor? How do we create them? Learn the basics of what actors are, how they work, and try out an actor yourself right on the Apify platform!
+menuWeight: 5.3
+paths:
+- apify-platform/actors
+---
+
+# [](#actors) Actors
+
+After you've followed the **Getting started** and **Apify CLI** lessons, and familiarized yourself with the Apify SDK in [**Web scraping for beginners**]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}}), you're almost ready to start creating actors! But before we get into that, let's discuss what an actor is, and a bit about how they work.
+
+## [](#what-is-an-actor) What's an actor?
+
+When you deploy your script to the Apify platform, it is then called an **actor**, which is simply a serverless microservice that accepts an input and produces an output. Actors can run for a few seconds, hours or even infinitely. An actor can perform anything from a simple action such as filling out a web form or sending an email, to complex operations such as crawling an entire website and removing duplicates from a large dataset.
```

Comment:
might be worth adding some links or explanations for serverless and microservice